package.json

```diff
@@ -127,8 +129,7 @@
     "stylelint-processor-styled-components": "1.10.0",
     "temp": "0.9.4",
     "ts-jest": "26.5.6",
-    "tsconfig-paths-webpack-plugin": "3.5.1",
     "typescript": "3.9.10",
-    "webpack": "4.46.0"
+    "webpack": "^5.58.0"
```

Comment:
I believe root dependencies in this project should be pinned, since they are managed by Renovate?